src/status_im/ui/screens/communities/community.cljs

```diff
@@ -256,7 +256,7 @@
 (defn unknown-community
   [community-id]
   (let [fetching (rf/sub [:communities/fetching-community community-id])]
-    [:<> {:style {:flex 1}}
```

Comment:
yeah thats weird :)

Reply:
It was sitting there and throwing errors in the console. Just cleaned it up :)